### ISSUE TITLE
Screen / font resize on Pixel 4a?

### ISSUE THREAD

**Issue by jason-mehmel** [bilgincoskun/brogue-android-port#11]
Happy to see that this exists!

The base font is pretty tiny on a Pixel 4a; I was hoping I could change it so that all the menu options and such would be easier to click. When I change the _custom_cell_width_ or _custom_cell_height_, it crashes and says:

**Font Error**
_resolution/cell size is too small for the minimum allowed font size_

It then closes and I need to uninstall and reinstall the APK to get it to work. (If I try loading it otherwise, I get the same error and closes.

Another issue here noted that the 4a menu was too small, and suggested resizing the screen, but I can't tell which other options are meant to resize the screen!

custom_screen_width and custom_screen_height don't see to change much when I try!

Any other options to make the font/buttons larger?

(To be really clear, this isn't about zooming on the character and map, but the overall UI itself.

**Comment by bilgincoskun** [comment]
For custom_cell_width / custom_cell_height, what was the values you have tried? They are defined in pixels, so can you try values like >100? Custom screen dimensions are the same, they need to have values like >1000x1980 etc.. By the way,instead of uninstalling you can just remove the config file in Brogue folder. 

You can resize/move the pad. But unfortunately menus and their text directly come from the game logic, they cannot be resized independently. The solution is moving menu logic to port but I doubt it will happen anytime soon since I am not actively developing, only maintaining the port right now.

**Comment by jason-mehmel** [comment]
Thank you for this!

I did just try values higher than 100 on cell width and height... and it
made everything so big all I could see was part of the gear 'options' icon!
But at least it didn't crash.

If I change the screen dimensions, would that also make the visual area
expand past the screen? (Hmmm. Maybe if I went the other way, changed the
dimensions to something of less resolution than the pixel 4a screen but in
the same ratio? I'll try that and report back.

I guess what I'm trying to figure out is if I can resize the 'font' without
expanding things beyond the phone screen... just to make the options a bit
more tappable with a finger, and easier to read as well.

I understand about not being able to move the menu logic! Thank you for the
work of maintaining the port as it is!

~Jason
pronouns: he / him / his
***@***.*** | www.jasonmehmel.com

**Comment by bilgincoskun** [comment]
Yeah I might have exaggerated cell width/height number. 
Cell and screen dimensions are basically for devices do not report usable screen size correctly, I doubt it will help in this case.

The port already tries to utilize the screen fully by selecting the biggest font size can fit into a cell. Short term solutions can be providing a custom font fills the cell by sacrificing readability: (https://github.com/bilgincoskun/brogue-android-port#graphics), or maybe adding an override setting for font/padding. I'll look into these options, although unfortunately not any time soon.

**Comment by timoxley** [comment]
Yeah +1 this, I keep mis-clicking things because of the small buttons, makes the game more stressful than it already is.
